delivery/http/routing/userrouter.go

```diff
@@ -3,18 +3,19 @@ package routing
 import (
 	"encoding/json"
 	"errors"
+	"net/http"
+
 	"github.com/go-playground/validator/v10"
 	"github.com/gorilla/context"
 	"github.com/gorilla/mux"
-	"kanca-studio/palang/delivery/http/schema"
-	"kanca-studio/palang/manager"
-	"net/http"
+	"github.com/kanca-studio/palang/delivery/http/schema"
```

Comment:
What's different between using `github.com` and not? While in `go.mod` we already stated the package name.

Reply:
Just for clarity where the source location, I'll delete it if prefer without the host